pkg/scaffold/olm-catalog/csv.go

```diff
@@ -357,3 +372,19 @@ func (s *CSV) updateCSVFromManifestFiles(cfg *CSVConfig, csv *olmapiv1alpha1.Clu
 
 	return store.Apply(csv)
 }
+
+func (s *CSV) setCSVCreationTimestamp(csv *olmapiv1alpha1.ClusterServiceVersion) error {
+	// Always use the timestamp from the CSV being updated, if either exist.
+	baseCSV, exists, err := getCSVFromFSIfExists(s.getFS(), s.getCSVPath(s.CSVVersion))
+	if err != nil {
+		return err
+	}
+	t := metav1.NewTime(time.Now())
+	if exists {
+		if tmp := baseCSV.GetCreationTimestamp(); tmp != (metav1.Time{}) {
```

Comment:
Should we at least log if CSV existed but timestamp was empty? As we are adding a creation timestamp on an already existing CSV. WDYT?